package.json

```diff
@@ -32,9 +32,10 @@
     "crypto": false,
     "stream": false
   },
+  "//": "@eluvio/elv-client-js:file://Users/todd/git/elv-client-js",
```

Comment:
what is this line for?

Reply:
oh sorry, it's just the only way to put a comment in a packge.json, will remove